src/include/common/re2_regex.h

```diff
@@ -0,0 +1,69 @@
+#pragma once
```

Comment:
As we discussed, let's remove our wrapper for re2 and use re2 functions directly in our codebase.